docs/build/guides/archival/restore-data-js.mdx

```diff
@@ -4,6 +4,12 @@ hide_table_of_contents: true
 description: Restore archived persistent contract data using the JavaScript SDK
 ---
 
+:::info
+
+The manual operation (`RestoreFootprintOp`) is (for the most part) no longer needed starting in Protocol 23, as archived entries are automatically restored when they appear in a footprint during `InvokeHostFunctionOp`. These examples are left here for historical context and for advanced use cases.
```

Comment:
Same as another the previous doc, I think historical context is a bit misleading. Since we have to specify the use-cases twice, we could list them in the state-archival.md doc and link them in these info sections.